src/IpScanner/Program.cs

```diff
@@ -5,18 +5,26 @@ namespace IpScanner;
 
 internal static class Program
 {
-    private static bool IsDisplayUnknownHost => false; // TODO: add ability for change from args
+    private static bool IsDisplayUnknownHost = false;
 
     private static Action<string?> Print { get; } = Console.WriteLine;
 
     static void Main(string[] args)
     {
         try
         {
-            if (args.Length != 2)
+            if (args.Length <= 2)
                 throw new Exception("Need two arguments: <Start IP> <End IP>");
 
-            var startIp = IPAddress.Parse(args[0]);
+			if (args.Length == 3)
+			{
+				var isValidFlag = char.TryParse(args[2], out var isDisplayUnknownHost);
+				if (!isValidFlag || isDisplayUnknownHost != 'e')
+					throw new Exception("Invalid flag found.");
```

Comment:
I like this one more to be honest.

Reply:
please describe why?

Reply:
well it put over head to remember. but I do agree 'e' does not make much sense and using some thing explicate make more sense. if you check some industry tools like nmap they use similar character as flags. it would be helpful we can decided some thing small and meaning full but for now I do accept your opinion and also if you can come up with something less robust. here is my suggestion '-u' or '--uh'